queue/deliverer.go

```diff
@@ -187,7 +194,23 @@ func (pl *Deliverer) ResourceWorker(queue Queue, done <-chan bool) {
 	}
 }
 
-func processMessage(rfcLogMessage syslog.Message) (*logging.Resource, error) {
+func (pl *Deliverer) processFilters(resource *logging.Resource) bool {
+	if pl.manager == nil {
+		return false
+	}
+	for _, p := range pl.manager.Plugins() {
```

Comment:
Might have to consider ordering of plugins (right now order of execution most likely determined from file scan order of the plugin files)

Reply:
Yes, been thinking about this. Simplest is probably to sort the plugin names alphabetically so you can control the processing order using smart naming. Could probably also consider the soure path e.g. give priority to plugins coming in via `LOGPROXY_PLUGINDIR`. Probably something for an upcoming patch.